### PR TITLE
members: add searchbar with filters

### PR DESCRIPTION
**Pull request by jrcastro2**
image

image

### CODE REVIEW

File: invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+FilterLabel.propTypes = {
+  updateQueryState: PropTypes.object.isRequired,
+  currentQueryState: PropTypes.func.isRequired,
+  filter: PropTypes.array.isRequired,
+};
+
+class SortLabel extends Component {
+  render() {
+    const { sort, currentQueryState, updateQueryState } = this.props;
+    const sortValue = sort.charAt(0).toUpperCase() + sort.slice(1);
```
Comment (ntarocco) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js:124]: Is this to have the uppercase first letter? eventually you might have a `_` func for that

File: invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+  updateQueryState: PropTypes.object.isRequired,
+  currentQueryState: PropTypes.func.isRequired,
+  sortOptions: PropTypes.array.isRequired,
+  filters: PropTypes.object.isRequired,
+  searchBarPlaceholder: PropTypes.string,
+  customCmp: PropTypes.node,
+};
+
+SearchBarWithFilters.defaultProps = {
+  searchBarPlaceholder: "Search ...",
```
Comment (ntarocco) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js:242]: Should the placeholder be translated somehow?

**Review by ntarocco — APPROVED** [review]

File: invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js (hunk 1 above)
Comment (jrcastro2) [reply invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js:124]: Good point! Will be addresed in the next PR.

File: invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js (hunk 2 above)
Comment (jrcastro2) [reply invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js:242]: Yes, thanks! Will be addresed in the next PR.

File: invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js (hunk 1 above)
Comment (jrcastro2) [reply invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js]: addressed

File: invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js (hunk 2 above)
Comment (jrcastro2) [reply invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js]: addressed

**Review by zzacharo — APPROVED** [review]

**Pull request merged**